src/GraphQL/Resolver/SectionResolver.php

```diff
@@ -23,8 +25,12 @@ public function __construct(SectionService $sectionService)
         $this->sectionService = $sectionService;
     }
 
-    public function resolveSectionById($sectionId)
+    public function resolveSectionById($sectionId): ?Section
     {
-        return $this->sectionService->loadSection($sectionId);
+        try {
+            return $this->sectionService->loadSection($sectionId);
+        } catch (UnauthorizedException $ex) {
```

Comment:
```suggestion
        } catch (UnauthorizedException $e) {
```

Reply:
Null, will allow you to display the sub-items list but without the section and without the error that is currently going to the logs

Reply:
That's fine, but how is this related to this comment? ;)